### PR TITLE
node and case study css. patches files

### PR DESCRIPTION
**Pull request by AJO-Python**
Adds patches to composer.json
Adds patches directory
Removes deprecated set_message() function

Adds sitewide "blue banner" title
Redesign node pages according to site redesign docs
Adjust case study page based on redesign doc
Adjust footer colour
Add coloured rows to "about" page

### CODE REVIEW

File: modules/custom/rii_forms/src/Form/InfoRequestForm.php [hunk 1]
```diff
@@ -42,7 +42,7 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
     $node->set("field_reason_for_info_request_re", $form_state->getValue('reason'));
     $node->field_q_6a->locality = "";  //Geocoder breaks if the address field has a region(locality) set
     $node->save();
-    drupal_set_message(t("Info request sent."));
+    // drupal_set_message(t("Info request sent."));
```
Comment (akademy) [inline modules/custom/rii_forms/src/Form/InfoRequestForm.php:45]: Can we add a Todo comment on where these are commented out please.

File: sites/default/files/css_editor/bootstrap.css [hunk 2]
```diff
@@ -10,9 +10,14 @@ first foreground:#581074;
     font-size: 18px;
     font-family: Arial, roboto, serif;
 }*/
-body{
-    overflow-x:hidden;
+
+// infrablue:
```
Comment (akademy) [inline sites/default/files/css_editor/bootstrap.css:14]: This isn't a valid comment in CSS. Should be /* */

Comment (akademy) [reply sites/default/files/css_editor/bootstrap.css:14]: A few other places this is used too.

File: sites/default/files/css_editor/bootstrap.css [hunk 3]
```diff
@@ -983,74 +1061,88 @@ p.support-text {
...
-  	border-radius: 0px 0px 0px 0px;
-  	padding-left: 1em;
-  	padding-top: 1em;
+
+.field--name-field-case-study-subtitle{
+  background-color: #03396C;
+  font-size: 1.5em;
+  text-align: left;
+  color: #FFFFFF;
+  border-radius: 0px 0px 0px 0px;
```
Comment (akademy) [inline sites/default/files/css_editor/bootstrap.css:1085]:
```suggestion
  border-radius: 0;
```
Could just be this, but not to bothered about stuff like that. Just FYI.

File: sites/default/files/css_editor/bootstrap.css [hunk 4]
```diff
@@ -1375,14 +1474,35 @@ html.js fieldset.collapsible.collapsed span.fieldset-legend {
...
+    flex:1 1 70% !important;
 }
+
 .rii .layout__region--second{
-    flex: 1 1 33% !important;
+    flex: 1 1 30% !important;
     text-align:center;
     margin-left:auto;
-}
+    position: absolute:
```
Comment (akademy) [inline sites/default/files/css_editor/bootstrap.css:1489]:
```suggestion
    position: absolute;

```

Comment (akademy) [reply sites/default/files/css_editor/bootstrap.css:1489]: Worth mentioned: When you "fix" a typo in a css file, it can complete screw up the styling!

**Review by akademy — COMMENTED** [review]
A few comments but fine.

File: sites/default/files/css_editor/bootstrap.css (hunk 4 above)
Comment (akademy) [reply sites/default/files/css_editor/bootstrap.css:1489]: This button that looks like the plus and minus in a page icon.

Please, feel free to remove it.

**Review by AJO-Python — COMMENTED** [review]
All comments actioned and added.
Merge is ready if you approve

**Pull request merged**